### ISSUE TITLE
Use TrueHTML to avoid mXSS

### ISSUE THREAD

**Issue by mishak87** [vojtech-dobes/nette.ajax.js#88]
**TL;DR not even assumed sanitized HTML and CSS can be trusted in `el.innerHTML` or `$(el).html()`.** Increase security by using JavaScript sanitizer like [Google Caja](https://code.google.com/p/google-caja/wiki/JsHtmlSanitizer).

More info:
http://www.nds.rub.de/research/publications/mXSS-Attacks/
http://www.slideshare.net/x00mario/the-innerhtml-apocalypse

Original issue https://github.com/nette/nette/issues/1496

---

I would suggest delegating [`$(el).html(snippet)`](https://github.com/vojtech-dobes/nette.ajax.js/blob/master/nette.ajax.js#L438-L445) to separate method and by default using secure implementation via TrueHTML or at least sanitizing the input.
**I haven't done enough research on this topic and futher research is necessary.** Known affected are old versions of IE (8, 9), Firefox, Chrome. IE 11 XSS filter shows warning even for false positives which might break trust between user and site.
